Docs: Guide: Get your Tequila API key (#572)
kiwicom/margarita: docs/guide-tequila-api-key.md

---
title: Guide: Get your Tequila API key
sidebar_label: Guide: Get your Tequila API key
---

> Margarita fetches data from GraphQL server. This server is just a proxy of officially supported Tequila REST API. The REST API requires an **API key** to be included in every call. GraphQL server loads the value from an environment variable named `API_KEY`.

Reviewer: ```suggestion
Margarita fetches data from the GraphQL server under `apps/graphql`. This server is just a proxy of officially supported Tequila REST API. The REST API requires an **API key** to be included in every call. The GraphQL server loads the value from an environment variable named `API_KEY`.
```

> To get your **API key** you will need to create an account on the [Tequila](tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.

Reviewer: ```suggestion
To get your **API key** you will need to create an account on the [Tequila](https://tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.
```

After successfully signing in to your account, you can read more information about **Tequila** and also explore the standard REST API.

> To get your **API key** for the GraphQL got into the [My applications](https://tequila.kiwi.com/portal/my-applications) page.

Reviewer: ```suggestion
To get your **API key** for the GraphQL server, go to the [My applications](https://tequila.kiwi.com/portal/my-applications) page.
```

> Create the new app, by clicking on the `add application` button.

Reviewer: ```suggestion
Create a new app by clicking on the `add application` button.
```

> Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.

Reviewer: ```suggestion
You should then choose the type of partnership you want with <span>Kiwi</span>.com: either **<span>Kiwi</span>.com Affiliate Program** partnership or **Book with <span>Kiwi</span>.com**  partnership. A description of these partnerships is available during this step.
```

> After choosing your partnership, you need to select the product type. Currently, the GraphQL proxy supports only _Search_ and _Search & Book_ features. The _Multicity_ and _Nomad_ search will be supported later.

Reviewer: ```suggestion
After choosing your partnership, you need to select the product type. Currently, the GraphQL proxy supports only _Search_ and _Search & Book_ features. The _Multicity_ and _Nomad_ search will be supported at a later date.
```

---
title: Guide: Get your Tequila API key
sidebar_label: Guide: Get your Tequila API key
---

> For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.

Reviewer 2: Margarita fetches data from GraphQL server. This server is just a proxy of officially supported Tequila REST API. The REST API requires an API key to be included in every call. GraphQL server loads the value from an environment variable named `API_KEY`.

Author: done

To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)

> . You can register and log in on the tequila.kiwi.com page.

Reviewer 2: To get your **API key** you will need to create an account on the [Tequila](tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.

Author: done

> After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.

Reviewer 2: After successfully signing in to your account, you can read more information about **Tequila** and also explore the standard REST API.

Author: done

To get your **API key** for the GraphQL got into the My applications page.

> Create the new app, by clicking on the `add application` button.

Reviewer 2: To get your **API key** go to the My Applications page and create the new app, by clicking on the `Add Application` button.

> go to the My Applications page

Can the "My Application page" be a link?

Author: done

> Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.

Reviewer 2: Why the `<span>` tags?

> More information about partnership types, you will find there.
Where? I'd probably remove this sentence if the info is available on the page.

Author: Because without this "hack", it behaves like a link

Author: done

> After choosing your partnership, you need to select the product type, our GraphQL proxy support only the _Search_ and _Search & Book_ features, but the _Multicity_ and _Nomad_ are on our roadmap and will come later.

Reviewer 2: After choosing your partnership, you need to select the product type. Currently, the GraphQL proxy supports only _Search_ and _Search & Book_ features. The _Multicity_ and _Nomad_ search will be supported later.

Author: done

> When you have been created application you will be redirected to the application detail page where you can grab your key from the Detail section.

Reviewer 2: Once you successfully created the application you will be redirected to the application detail page where you can grab your **API key** from the Detail section.

Author: done
